packages/@webex/internal-plugin-metrics/src/utils.ts

```diff
@@ -15,3 +21,66 @@ export const generateCommonErrorMetadata = (error) => {
 
   return error;
 };
+
+/**
+ * Returns the deviceId from our registration with WDM.
+ * @returns {string} deviceId or empty string
+ */
+export function getDeviceId(): string {
+  const {url} = this.device;
```

Comment:
this will break, the previous implementation relies on this.webex.internal.device which can only be accessed inside a plugin

Reply:
Yeah - as for the `this.version` below. I need to fix that.